judoscale-rails/lib/judoscale/rails/config.rb

```diff
@@ -3,12 +3,15 @@
 module Judoscale
   module Rails
     module Config
-      attr_accessor :start_reporter_after_initialize, :rake_task_ignore_regex
+      attr_accessor :start_reporter_after_initialize, :rake_task_ignore_regex, :utilization_enabled, :utilization_interval
```

Comment:
My first implementation had utilization be a config object so you could do `utilization.enabled` and `utilization.interval`, but I bailed on that to keep it simpler.